Quadriga/src/main/resources/locale/messages_en_US.properties

```diff
@@ -218,3 +218,11 @@ filename.proper = The filename should only contain the following characters: 0-9
 reference.required = A reference is required. Please provide the URI of the original copy of the text.
 empty.textfiles = There are no text files in this workspace.
 accessibility.required = Who should be able to see this text? Everybody (select "Public") or just members of your project (select "Private")?
+
+
+#Resolver Validation error messages
+resolver.validate_success = The regular expression pattern you provided was successfully applied to the provided handle examples.
+resolver.validate_failure =  The regular expression pattern you provided could not be applied to the provided handle examples.
+resolver.delete_failure = The given resolver cannot be deleted as it is being used by Project.
```

Comment:
incorrect English. Should be: "... as it is still being used by at least one project."